### PR TITLE
M3-1802 Unhumanize Backups Datestamp

### PR DESCRIPTION
**Pull request by martinmckenna**
## Description

Use the actual ISO-8601 timestamp instead of the humanized time for backups rows

## Type of Change
- Non breaking change ('update', 'change')

## Note to Reviewers

Moved over all non-src directory code to devDependencies. TIL that yarn/npm makes no distinction between the two - Non-src dir code wasn't being included in the build, regardless of whether it's in the dependencies

ALSO I changed the `formatDate` function a bit. I made the function get the timezone from redux store itself, rather than having to pass it as an argument

### CODE REVIEW

File: src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx [hunk 1]
```diff
@@ -479,8 +479,8 @@ class LinodeBackup extends React.Component<CombinedProps, State> {
           <Table aria-label="List of Backups">
             <TableHead>
               <TableRow>
-                <TableCell>Date Created</TableCell>
                 <TableCell>Label</TableCell>
+                <TableCell>Date Created</TableCell>
```
Comment (johnwcallahan) [inline src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx]: Was the order-switching of "Label" and "Date Created" intentional?

File: src/utilities/formatDate.ts [hunk 2]
```diff
@@ -37,21 +40,25 @@ export const shouldHumanize = (
...
 export const formatDate = (
   date: string,
   options: FormatDateOptions = {}
-): string | null => {
+): string => {
   let time;
+
+  /** get the timezone from redux and use it as the moment timezone */
+  const reduxProfile = store.getState().__resources.profile;
+  const userTimezone = pathOr('GMT', ['data', 'timezone'], reduxProfile);
```
Comment (johnwcallahan) [inline src/utilities/formatDate.ts]: Good call getting this from Redux.

File: src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx (hunk 1 above)
Comment (martinmckenna) [reply src/features/linodes/LinodesDetail/LinodeBackup/LinodeBackup.tsx]: that's what the AC in the ticket said so I guess so?

**Comment by martinmckenna** [comment]
@johnwcallahan leaving the landing the way it is, as per Jamie

**Pull request merged**